web/src/features/moreCast2/components/TabbedDataGrid.tsx

```diff
@@ -284,7 +284,7 @@ const TabbedDataGrid = ({ morecast2Rows, fromTo, setFromTo, fetchWeatherIndeterm
 
   useEffect(() => {
     const newVisibleRows: MoreCast2Row[] = []
-    const stationCodes = selectedStations.map(station => station.station_code)
+    const stationCodes = selectedStations?.map(station => station.station_code)
```

Comment:
Are these possibly undefined now? Maybe the react types changed?

Reply:
I was trying to debug unit tests which complained of .map being called in undefined and this got left behind as an artifact of my debugging.

I'll revert the change. Thanks!